src/components/button/Button.jsx

```diff
@@ -1,5 +1,13 @@
 import "./Button.scss";
+import cn from "classnames";
 
-export default function Button({text, type = "primary"}) {
-  return <button className={`button button--${type}`}>{text}</button>;
+export default function Button({ text, color }) {
+  const buttonColor = cn(
+    "button",
+    { "button--orange": color === "orange" },
+    { "button--green": color === "green" },
+    { "button--blue": color === "blue" }
+  );
+
+  return <button className={buttonColor}>{text}</button>;

```

Comment:
```suggestion
  return <button type="button" className={buttonColor}>{text}</button>;
```

Just for the accessibility best practices